components/google-cloud/google_cloud_pipeline_components/experimental/AutoML/create_dataset_for_tables/component.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import NamedTuple
+
+
+def automl_create_dataset_for_tables(
```

Comment:
This comment is for all AutoML components. should we be moving the old GCP AutoML components to deprecated folder instead?

Reply:
yeah agreed these are quite old services. 
though it doesn't hurt to keep them just in case people still need them, since these services are not deprecated.